deploy/AzureAppServices/Datadog.Development.AzureAppServices.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Datadog.Development.AzureAppServices</id>
+    <version>1.10.40-prerelease</version>
+    <title>Unstable .NET Datadog APM</title>
+    <authors>Datadog</authors>
+    <license type="expression">Apache-2.0</license>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+      Unstable Development Datadog .NET Tracer
```

Comment:
Why does this say `Unstable`? Wouldn't `Internal` or some other more neutral term be better?

Reply:
Agreed, `Unstable` might deter people from using it

Reply:
Yes the point is to not have people use this.

Reply:
How about "experimental?"